packages/nextjs/app/migration/about.tsx

```diff
@@ -60,14 +60,19 @@ const AboutPage: NextPage = () => {
         </div>
         <div className="order-2">
           <NextImage
+            alt="big picture diagram"
             src={require("../../../../docs/img/big-picture.png")}
             loader={localImageLoader}
-            layout="intrinsic"
             width={DIAGRAM_WIDTH}
             height={DIAGRAM_HEIGHT}
-            objectFit="contain"
-            objectPosition="center"
             className="rounded-2xl"
+            priority
+            style={{
```

Comment:
Can we do these with Tailwind instead of style objects?

Reply:
Sure can! 👍

Reply:
Nice! Looks great.